internal/blocktx/health_check_test.go

```diff
@@ -52,10 +52,10 @@ func TestCheck(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			// given
 			storeMock := &storeMocks.BlocktxStoreMock{
-				GetBlockGapsFunc: func(ctx context.Context, heightRange int) ([]*store.BlockGap, error) {
+				GetBlockGapsFunc: func(_ context.Context, _ int) ([]*store.BlockGap, error) {
```

Comment:
In most cases I don't like unused parameters, so it's nice to mark them with underscore, but sometimes they are useful in telling us what the function takes and what we can get from these parameters. In this case, I would prefer these parameters to stay named, even tho we don't use them. WDYT?

Reply:
For me both is fine. Also here @pawellewandowski98 is it possible to configure the revive so this rule is not enforced?